mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraCurrentQuotaManager.java

```diff
@@ -89,16 +93,16 @@ public QuotaCountUsage getCurrentMessageCount(QuotaRoot quotaRoot) throws Mailbo
     }
 
     @Override
-    public QuotaSizeUsage getCurrentStorage(QuotaRoot quotaRoot) throws MailboxException {
+    public QuotaSizeUsage getCurrentStorage(QuotaRoot quotaRoot) {
         ResultSet resultSet = session.execute(getCurrentStorageStatement.bind(quotaRoot.getValue()));
         if (resultSet.isExhausted()) {
             return QuotaSizeUsage.size(0L);
         }
         return QuotaSizeUsage.size(resultSet.one().getLong(CassandraCurrentQuota.STORAGE));
     }
 
-    private void checkArguments(long count, long size) {
-        Preconditions.checkArgument(count > 0, "Count should be positive");
-        Preconditions.checkArgument(size > 0, "Size should be positive");
+    private void checkArguments(CurrentQuota currentQuota) {
+        Preconditions.checkArgument(currentQuota.getCount().asLong() > 0, "Count should be positive");
+        Preconditions.checkArgument(currentQuota.getSize().asLong() > 0, "Size should be positive");
```

Comment:
We should not be able to build an invalid `CurrentQuotaDiff` in the first place...